eureka2-core/src/main/avro/discovery-schema.avdl

```diff
@@ -74,6 +74,21 @@ protocol DiscoveryProtocol {
         } deltaDTO;
     }
 
+    @namespace("com.netflix.eureka2.interests.StreamStateNotification$")
+    enum BufferingState {
+        Unknown, Buffer, FinishBuffering
```

Comment:
A bit pedantic, can we shorten the name here to something like Buffer and EndBuffer?